test/SILGen/serialize_module_for_stored_property_initializer.swift

```diff
@@ -0,0 +1,47 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+// RUN: %target-swift-frontend -c %t/inlinedStruct.swift -static -O -parse-as-library -module-name InlinedStructs -emit-module-path %t/InlinedStructs.swiftmodule -o  %t/inlinedStruct.swift.o
+// RUN: %target-swift-frontend -c -emit-ir -O  %t/use.swift -I %t -o %t/use.swift.ir
+// RUN: cat %t/use.swift.ir | %FileCheck %s
+
+//--- inlinedStruct.swift
+
+@usableFromInline
+struct CMSSignedData {
+    @usableFromInline var field: Bool?
+
+    @inlinable
+    init(field: Bool?) {
+        self.field = field
+    }
+}
+
+public struct TestS {
+  @usableFromInline
+  let x: CMSSignedData = CMSSignedData(field: false)
+
+  @inlinable
+  public init() {  }
+
+   @inlinable
+  public var field: Bool {
+    return x.field!
+  }
+}
+
+//--- use.swift
+
+import InlinedStructs
+
+public struct TestTwo {
+  let field: TestS = TestS()
+}
+
+public func testTwo() -> Bool {
+  let x = TestTwo()
+  return x.field.field
+}
+
+// Ensure that the variable initialization expression is not dllimported on Windows.
+// CHECK: declare swiftcc i8 @"$s14InlinedStructs5TestSV1xAA13CMSSignedDataVvpfi"()
```

Comment:
Might be good to have two separate tests, one with `-static` and one without `-static` to ensure that we generate it correctly in both cases.

Reply:
good idea!